telegrambots-abilities/src/main/java/org/telegram/abilitybots/api/bot/BaseAbilityBot.java

```diff
@@ -660,7 +660,7 @@ private void updateUserId(User oldUser, User newUser) {
 
     boolean filterReply(Update update) {
         return replies.stream()
-                .filter(reply -> runSilently(() -> reply.isOkFor(update), reply.name()))
```

Comment:
The title of this PR is misleading. You are indeed removing the function which is no longer required, but you're changing the behavior of this pipeline.

The reply must be checked, its flags (Predicates of type Predicate<Boolean>) pass.